scripts/monitoring/cron-send-usage-pv.py

```diff
@@ -0,0 +1,141 @@
+    logger.debug("parse_args()")
+
+    parser = argparse.ArgumentParser(description='OpenShift pv usage ')
+    parser.add_argument('-v', '--verbose', action='store_true', default=None, help='Verbose?')
+    parser.add_argument('--namespace', default="openshift-infra",
+                        help='namespace (be careful of using existing namespaces)')
+    return parser.parse_args()
+
+def send_metrics(usage,capacity,used):
+    """ send data to MetricSender"""
+    logger.debug("send_metrics()")
+    ms_time = time.time()
+    ms = MetricSender()
+    logger.info("Send data to MetricSender")
+    ms.add_metric({'openshift.master.pv.percent.usage': usage})
+    ms.add_metric({'openshift.master.pv.capacity.max': capacity})
+    ms.add_metric({'openshift.master.pv.capacity.used': used})
+    ms.send_metrics()
+    logger.info("Data sent to Zagg in %s seconds", str(time.time() - ms_time))
+
+def get_dynamic_pod_name():
+    """get the max_capacity of the cluster"""
+    pods = ocutil.get_pods()
+    dynamic_pod_name = ""
+    pattern = re.compile(r'online-volume-provisioner-') 
+    for pod in pods['items']:
+        pod_name = pod['metadata']['name']
+        match = pattern.search(pod_name)
+        #find the dynamic pod name
+        if match:
+            dynamic_pod_name = pod_name
+    return dynamic_pod_name
+
+def get_max_capacity(pod_name):
+    """get the env from the dynamic pod """
+    env_info = runOCcmd_yaml(" env pod/"+pod_name)
+
+    envs = env_info['spec']['containers']
+    max_capacity = 0
+    for en in envs[0]['env']:
+        if en['name'] == 'MAXIMUM_CLUSTER_CAPACITY':
+            max_capacity = en['value']
+    return max_capacity
+
+def get_pv_usage():
+    """get all the pv used """
+    pv_info = runOCcmd_yaml(" get pv")
+    total = 0
+    for pv in pv_info['items']:
+        namespace = pv['spec']['claimRef']['namespace']
+        capacity = pv['spec']['capacity']['storage']
+        if namespace != "openshift-infra" and namespace != "logging":
+            capacity_int = int(capacity.replace('Gi',''))
+            total = total + capacity_int
+    return total
+def main():
+    """ report pv usage  """
+    exception = None
+
+    logger.info('################################################################################')
+    logger.info('  Starting Report pv usage - %s', datetime.datetime.now().strftime("%Y-%m-%d %H:%M"))
+    logger.info('################################################################################')
+    logger.debug("main()")
+
+    args = parse_args()
+    if args.verbose:
+        logger.setLevel(logging.DEBUG)
+    ocutil.namespace = "openshift-infra"
+    dynamic_pod_name = get_dynamic_pod_name()
+    if dynamic_pod_name == "":
+        #means this cluster have no dynamic pv 
+        pass
+    else:
+        MAXIMUM_CLUSTER_CAPACITY = get_max_capacity(dynamic_pod_name)
+        logger.debug(MAXIMUM_CLUSTER_CAPACITY)
+	pv_used = get_pv_usage()
```

Comment:
@BlueShells does this line fail? it doesn't seen to have the correct whitespace .. (add one space)